Makefile

```diff
@@ -1,8 +1,12 @@
+SHELL = bash
 GOCQL_REPO ?= github.com/scylladb/gocql
 DOCKER_IMAGE_TAG ?= scylla-bench
 GOOS ?= $(shell uname | tr '[:upper:]' '[:lower:]')
 GOARCH ?= $(shell go env GOARCH)
+MAKEFILE_PATH := $(abspath $(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
+BIN_DIR := "${MAKEFILE_PATH}/bin"
 
+GOLANGCI_VERSION = 2.6.0
```

Comment:
the downside of this here, we won't get renovate updates about it.

again, golang doesn't have test dependencies ?

Reply:
I would recommend doing it as a followup

Reply:
@fruch we are already using `tools` from go 1.24 and its facing issue with golangci-lint for some reason, it does not want to be installed as a "library".

The only way to have something similar to the setup that we have right now is to make a seperate go module (tools module) and import it from there as @dkropachev is suggesting, but that is also prone to breaking, so i'd leave it as is, and not worry about renovate, since golangci-lint is making so many changes and breaking CI, this needs to be updated manually anyways, better to keep it as is from this PR